packages/app/src/app/pages/common/Modals/StorageManagementModal/FilesList/index.tsx

```diff
@@ -68,9 +68,9 @@ export const FilesList: FunctionComponent = () => {
                   width: 40%;
                 `}
               >
-                <CheckBox
-                  onClick={() => toggleCheckbox(id)}
-                  selected={selectedItems.includes(id)}
+                <Checkbox
+                  onChange={() => toggleCheckbox(id)}
+                  checked={Boolean(selectedItems.includes(id))}
```

Comment:
`includes()` already returns a `boolean`

```suggestion
                  checked={selectedItems.includes(id)}
```